scripts/globals/shop.lua

```diff
@@ -383,5 +403,157 @@ tpz.shop =
                 8828,    7600,      tpz.craftRank.AMATEUR,      -- Smithing Kit 50
                 9247, 1126125,      tpz.craftRank.AMATEUR       -- Niobium Ore
         }
+    },
+
+    curioVendorMoogleStock =
+    {
+        ["medicine"] =
```

Comment:
quote-string index keys?

Reply:
something wrong with it @TeoTwawki ?

Reply:
It can be done either way, as long as you're indexing with `[ ]` though it isn't strictly required unless you're using words which aren't proper identifiers... I'm personally okay with it either way.

Reply:
I'd kinda like to keep with the existing style rather than having one thats just different without needing to be

Reply:
Done @TeoTwawki  please give it your blessing.
Just a heads-up in the if statement, I made it nested so whomever takes ownership of keyitems can just add the extra option for it #notIt lol

Reply:
Ima preist now, I bless PRs.

:latin_cross: